install_config/cluster_metrics.adoc

```diff
@@ -600,25 +600,38 @@ To avoid this issue, use certificates which are configured to be acceptable by
 your browser.
 ====
 
-ifdef::openshift-origin[]
 == Accessing Hawkular Metrics Directly
 
 To access and manage metrics more directly, use the Hawkular Metrics API.
 
+[NOTE]
+====
+When accessing Hawkular Metrics via the API you will only be able to perform
+reads, writing metrics has been disabled by default.  If you wish for
+individual users to also be able to write metrics, you will need to set the
+'*USER_WRITE_ACCESS*' deployer template parameter to 'true'.
+
+It is recommended to disable user write access and only have metrics enter the
+system via Heapster.  If write access is enabled, any user will be able to
+write metrics to the system which can affect performance and cause Cassandra
+disk usage to unpredictably increase.
+====
+
 The link:http://www.hawkular.org/docs/rest/rest-metrics.html[Hawkular Metrics
 documentation] covers how to use the API, but there are a few differences when
 dealing with the version of Hawkular Metrics configured for use on
 {product-title}:
 
```

Comment:
We need an anchor here as well, so:
[[cluster-metrics-openshift-projects-and-hawkular-tenants
=== {product-title} Projects and Hawkular Tenants

Also, spell out and instead of &